### ISSUE TITLE
Development branch - Wrong totals?

### ISSUE THREAD

**Issue by margarett** [OpenImporter/openimporter#57]
In function substep30() totals and last message/topic/member are fetched. But unapproved members or posts should not really be counted towards totals... But eg, Elk, already moved the "approved" out of "members" table, so there isn't really a common way to do that, am I right?

**Comment by emanuele45** [comment]
Good point.
I think you mean ```is_activated``` for members. If so, Elk didn't move it out, neither did move ```approved``` from messages. ;)

This actually applies to both branches.
I hope recount statistics fixes it... or not?

**Comment by margarett** [comment]
You are right, is_activated is there, I don't know what the heck I was looking at yesterday evening :open_mouth: 

Recount statistics does fix it ;-)

**Comment by emanuele45** [comment]
Too much alcohol... or not enough. :stuck_out_tongue: 

I guess a couple of conditions in the query should do the trick.

**Comment by margarett** [comment]
WHERE is_activated = 1 :question:

**Comment by emanuele45** [comment]
is_activated is a darn beast:
https://github.com/elkarte/Elkarte/wiki/is_activated (that is valid for SMF as well)
I **think**, but I may very well be totally wrong, that 1 and 2 are counted as members in the total and I feel that also 4 is helping bumping up the number.
